packages/palette/src/elements/ModalDialog/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./ModalDialog"
+export * from "./ModalDialogBackdrop"
+export * from "./ModalDialogDialog"
```

Comment:
This is modular so that you could, for example, swap in a new backdrop or custom dialog.